nemo/collections/asr/modules/conformer_encoder.py

```diff
@@ -87,9 +88,13 @@ class ConformerEncoder(NeuralModule, StreamingEncoder, Exportable, AccessMixin):
             Defaults to 5000
         n_heads (int): number of heads in multi-headed attention layers
             Defaults to 4.
-        att_context_size (List[int]): List of 2 ints corresponding to left and right attention context sizes,
-            or None for full context.
-            Defaults to None.
+        att_context_size (list): specifies the context sizes on each side. Each context size should be a list of two integers like [100,100].
```

Comment:
Since we support both List[int] and List[list[init]], lets change it to `List[Union[List[int],int]]`

Reply:
done.